packages/excalidraw/components/ColorPicker/ColorPicker.tsx

```diff
@@ -106,7 +106,7 @@ const ColorPickerPopupContent = ({
   return (
     <PropertiesPopover
       container={container}
-      style={{ maxWidth: "208px" }}
+      style={{ maxWidth: "17em" }}
```

Comment:
```suggestion
      style={{ maxWidth: "13em" }}
```

208/16=13

With 17em, it's this wide:

![image](https://github.com/user-attachments/assets/cb02a4fe-40c9-4c72-a52c-e7a374f24bf3)